frontend/src/scenes/settings/organization/inviteLogic.ts

```diff
@@ -130,6 +138,24 @@ export const inviteLogic = kea<inviteLogicType>([
                 return invites.filter(({ level }) => level === OrganizationMembershipLevel.Owner).length > 0
             },
         ],
+        isAllowToSendInvites: [
```

Comment:
```suggestion
        isAllowedToSendInvites: [
```

will need to find&replace if you decide to make this change